python/ray/util/lightning_accelerators/examples/ptl_horovod_ray_example.py

```diff
@@ -0,0 +1,195 @@
+"""Example using Pytorch Lightning with a Horovod on Ray Accelerator."""
+import os
+import tempfile
+
+import pytorch_lightning as pl
+import torch
+from torch.utils.data import random_split, DataLoader
+from torchvision.datasets import MNIST
+from torchvision import transforms
+
+import ray
+from ray import tune
+from ray.tune.examples.mnist_ptl_mini import LightningMNISTClassifier
+from ray.tune.integration.pytorch_lightning import TuneReportCallback
+from ray.util.lightning_accelerators import HorovodRayAccelerator
+
+
+class MNISTClassifier(LightningMNISTClassifier):
+    def prepare_data(self):
+        self.dataset = MNIST(
+            self.data_dir,
+            train=True,
+            download=True,
+            transform=transforms.ToTensor())
+
+    def train_dataloader(self):
+        dataset = self.dataset
+        train_length = len(dataset)
+        dataset_train, _ = random_split(
+            dataset, [train_length - 5000, 5000],
+            generator=torch.Generator().manual_seed(0))
+        loader = DataLoader(
+            dataset_train,
+            batch_size=self.batch_size,
+            shuffle=True,
+            num_workers=1,
+            drop_last=True,
+            pin_memory=True,
+        )
+        return loader
+
+    def val_dataloader(self):
+        dataset = self.dataset
+        train_length = len(dataset)
+        _, dataset_val = random_split(
+            dataset, [train_length - 5000, 5000],
+            generator=torch.Generator().manual_seed(0))
+        loader = DataLoader(
+            dataset_val,
+            batch_size=self.batch_size,
+            shuffle=False,
+            num_workers=1,
+            drop_last=True,
+            pin_memory=True,
+        )
+        return loader
```

Comment:
Can we just use the ptl_bolts mnist module?

Reply:
The ptl bolts data module didn't work with multi-node training. The data downloading had to be done within the trainer so the data could be downloaded on each node.